godot-core/src/builtin/vectors/vector3.rs

```diff
@@ -38,42 +39,39 @@ pub struct Vector3 {
     pub z: real,
 }
 
-impl Vector3 {
-    /// Vector with all components set to `0.0`.
-    pub const ZERO: Self = Self::splat(0.0);
+impl_vector_operators!(Vector3, real, (x, y, z));
 
-    /// Vector with all components set to `1.0`.
-    pub const ONE: Self = Self::splat(1.0);
+impl_vector_consts!(Vector3, real);
+impl_float_vector_consts!(Vector3);
+impl_vector3x_consts!(Vector3, real);
 
-    /// Unit vector in -X direction. Can be interpreted as left in an untransformed 3D world.
-    pub const LEFT: Self = Self::new(-1.0, 0.0, 0.0);
+impl_vector_fns!(Vector3, RVec3, real, (x, y, z));
+impl_float_vector_fns!(Vector3, (x, y, z));
+impl_vector3x_fns!(Vector3, real);
+impl_vector2_vector3_fns!(Vector3, (x, y, z));
+impl_vector3_vector4_fns!(Vector3, (x, y, z));
 
-    /// Unit vector in +X direction. Can be interpreted as right in an untransformed 3D world.
-    pub const RIGHT: Self = Self::new(1.0, 0.0, 0.0);
+impl Vector3 {
+    /// Unit vector pointing towards the left side of imported 3D assets.
+    pub const MODEL_LEFT: Self = Self::new(1.0, 0.0, 0.0);
 
-    /// Unit vector in +Y direction. Typically interpreted as up in a 3D world.
-    pub const UP: Self = Self::new(0.0, 1.0, 0.0);
+    /// Unit vector pointing towards the right side of imported 3D assets.
+    pub const MODEL_RIGHT: Self = Self::new(-1.0, 0.0, 0.0);
 
-    /// Unit vector in -Y direction. Typically interpreted as down in a 3D world.
-    pub const DOWN: Self = Self::new(0.0, -1.0, 0.0);
+    /// Unit vector pointing towards the top side (up) of imported 3D assets.
+    pub const MODEL_UP: Self = Self::new(0.0, 1.0, 0.0);
```

Comment:
This is called `MODEL_TOP` in Godot. 

While "up" might be more accurate, we would also need to rename others to "down", "forward" and "backward", at which point it's easier to adopt Godot's terms.

Reply:
This is just an error by me and wasn't a deliberate change, sorry about that.

Reply:
All good :blush: